### DISCUSSION TITLE
Enabling ROCm 6.0.2 on ubuntu 22.04 for a 7900 xtx?

### DISCUSSION THREAD

**Discussion by rudiservo** [ROCm/ROCm#2955] [General]
Hey guys just wondering I have ROCm 6.0.2 installed, so apt says
but rocm-smi --version states

ROCM-SMI-LIB version: 6.0.0

I do realize that smi is not the same has the rocm core libraries, but I am kind of worried this might be an issue with the package management.

**Comment by taylding-amd** [comment]
Hi @rudiservo, you can find the corresponding ROCm SMI version in our [Compatibility Matrix](https://rocm.docs.amd.com/en/latest/compatibility/compatibility-matrix.html#). For ROCm 6.0.2, the appropriate ROCm SMI version is also 6.0.2. For installation instructions, please refer to the ROCm [SMI installation guide](https://rocm.docs.amd.com/projects/rocm_smi_lib/en/latest/install/install.html). Please feel free to reach out if you have any further questions!
